core/src/main/scala/akka/persistence/cassandra/journal/TagWriters.scala

```diff
@@ -179,8 +179,14 @@ import akka.util.ByteString
       tagActor(tw.tag).forward(tw)
     case BulkTagWrite(tws, withoutTags) =>
       tws.foreach { tw =>
-        updatePendingScanning(tw.serialised)
-        tagActor(tw.tag).forward(tw)
+        if (!currentPersistentActors.contains(tw.serialised.head.persistenceId)) {
```

Comment:
As I understand it, all `tw.serialised` belongs to the same `persistenceId` and therefore using `head` here is fine. Please confirm that thinking @chbatey

Reply:
:+1: